recipe/build.sh

```diff
@@ -5,7 +5,7 @@ set -x
 # Get an updated config.sub and config.guess
 cp $BUILD_PREFIX/share/gnuconfig/config.* .
 
-# The compiler flags interfere with th build and we need to overide them :-/
+# The compiler flags interfere with the build and we need to overide them :-/
```

Comment:
I think we could try removing this code that unsets the flags.

Reply:
Is the etiquette for me to do that or for you?